### DISCUSSION TITLE
difference between with_custom_head and render link

### DISCUSSION THREAD

**Discussion by kzhui125** [DioxusLabs/dioxus#1886] [Q&A] [answered]
Are they identical?

```rs
dioxus_desktop::launch_cfg(
  App,
  dioxus_desktop::Config::new()
    .with_custom_head(r#"<link rel="stylesheet" href="public/tailwind.css">"#.to_string()))
```

```rs
render!(
        link {
            rel: "stylesheet",
            href: "public/tailwind.css"
        }
```

Thanks,

**Comment by ealmloff** [comment, marked as answer]
In dioxus, you components all render into the `<body>` element which contains the content of the page. `<link>` elements should be placed in the `<head>` element instead which contains some metadata/resources for the page. The first code snippet correctly places it in the head, the second code snippet will place the link somewhere in the body

Some relevant discussions:
- Modifying the head in a cross platform way potentially with rsx https://github.com/DioxusLabs/dioxus/issues/1169
- Linking assets in rust code at compile time https://github.com/DioxusLabs/dioxus/issues/1283
